event_log.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-import event_pb2
+from tensorflow.core.util import event_pb2
```

Comment:
this is actually importing a different proto; i'm wanting to use the `event.proto` at the root of this repro.

it's a bit hacky as it is now; this code requires you run `protoc event.proto --python_out=.` (as mentioned in the README.md) before the `event_pb2.py` is created. what i should do is try the import and, if it fails, give a warning that this code needs to be run (either that or make it a proper part of some kind of `setup.py` init...